src/less/components/search/SearchFacade.less

```diff
@@ -0,0 +1,32 @@
+/*
+  Mixins to style <hx-search> facade states.
+*/
+
+.SearchFacade(pristine-focus) {
+  border: 1px solid @cyan-700;
+  box-shadow: @focus-glow;
+}
+
+.SearchFacade(invalid) {
+  border: 2px solid @red-900;
+}
+
+.SearchFacade(invalid-focus) {
+  border: 2px solid @red-900;
+  box-shadow: @focus-glow-invalid;
+}
+
+.SearchFacade(disabled) {
+  background-color: @gray-50;
+  border: 1px solid @gray-500;
+  color: @gray-400;
+}
+
+.SearchFacade(disabled-focus) {
+  border: 1px solid @gray-500;
+}
+
+.SearchFacade(button-focus) {
```

Comment:
Please add a `TODO` comment to rename/move this mixin, as it is not intended to be used on the `hx-search`, but a sibling `<button>` instead.

Reply:
Comment added:

```less
// TODO: Rename/move mixin
```